sdk/logs/src/main/java/io/opentelemetry/sdk/logs/export/BatchLogRecordProcessorBuilder.java

```diff
@@ -150,6 +163,10 @@ int getMaxExportBatchSize() {
    * @return a new {@link BatchLogRecordProcessor}.
    */
   public BatchLogRecordProcessor build() {
+    if (maxExportBatchSize > maxQueueSize) {
+      maxExportBatchSize = Math.min(DEFAULT_MAX_EXPORT_BATCH_SIZE, maxQueueSize);
+      logger.log(Level.FINE, "Using maxExportBatchSize: {0}", maxExportBatchSize);
```

Comment:
could we make this log indicate that we're overriding the value for `maxExportBatchSize` since it was larger than `maxQueueSize`?